pkg/components/standalone_loader.go

```diff
@@ -54,7 +53,11 @@ func (s *StandaloneComponents) LoadComponents() ([]components_v1alpha1.Component
 	list := []components_v1alpha1.Component{}
 
 	for _, file := range files {
-		if !file.IsDir() && s.isYaml(file.Name()) {
+		if !file.IsDir() {
+			if !utils.IsYaml(file.Name()) {
+				log.Warnf("A non-YAML component file %s was detected, it will not be loaded, for more detail, please refer to https://github.com/dapr/dapr/pull/4679", file.Name())
```

Comment:
Please, do not refer to GitHub. You can keep a simpler message now and then update once you have added documentation in the docs website.